students/darrell_h/lesson_4/mailroom.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/local/bin/python3
+
+d = {'jack': [100, 200, 300, 400],
+     'mary': [3000, 5000],
+     'frank': [29.50, 31],
+     'jane': [3000, 5000],
+     'scrouge': [1, 2, 3],
+     'bob': [60000, 70000, 7668, 4]}
+
+
+#####################
+#  HELPER FUNCTIONS #
+#####################
+
+def seperator(str):
```

Comment:
excellent use of helper functions.  When defining helper functions, we want to start them with an `_` which lets other programmers know they are helpers and not for use outside this python file (although there is nothing actually preventing them from being used outside the file)
```
def _seperator(str):

def _add_donation(donor_name, existing_donor):
```